src/docfx/Glob/GlobUtility.cs

```diff
@@ -28,7 +28,8 @@ public static FileMapping ExpandFileMapping(string baseDirectory, FileMapping fi
             var expandedFileMapping = new FileMapping();
             foreach (var item in fileMapping.Items)
             {
-                var src = Path.Combine(baseDirectory, item.SourceFolder ?? string.Empty);
+                string expandedSourceFolder = Environment.ExpandEnvironmentVariables(item.SourceFolder ?? string.Empty);
+                var src =  Path.IsPathRooted(expandedSourceFolder) ? expandedSourceFolder : Path.Combine(baseDirectory, expandedSourceFolder);
```

Comment:
The Path.IsPathRooted call should not be necessary. Path.Combine will detect if expandedSourceFolder is rooted and not let baseDirectory affect the return value in that case. <https://github.com/microsoft/referencesource/blob/74eb1593e09a636270482f1c0525aabdccb1f364/mscorlib/system/io/path.cs#L1295-L1296>

Reply:
you are right